src/components/ProfilePage/TextInput/TextInput.tsx

```diff
@@ -1,23 +1,21 @@
+/* eslint-disable react/require-default-props */
 /* eslint-disable react/jsx-props-no-spreading */
 import { Typography } from '@mui/material';
 import InputBase from '@mui/material/InputBase';
-import { UseFormRegister } from 'react-hook-form/dist/types/form';
+import { FieldError } from 'react-hook-form';
+import { UseFormRegisterReturn } from 'react-hook-form/dist/types/form';
 
 import styles from './TextInput.module.css';
 
-interface ErrorMessage {
-  message: string;
-}
-
 interface Props {
   placeholder: string;
-  register: UseFormRegister<Record<string, unknown>>;
+  register?: UseFormRegisterReturn;
   // eslint-disable-next-line react/require-default-props
```

Comment:
It seems to be no longer needed as you added the corresponding line at the top of this file.